### PR TITLE
Issue #4318: Theme settings: Fix color input label alignment

### LINKED ISSUE #4318: Theme color settings: color input not aligned with its label when label is too long
Before:

Screen Shot 2020-02-23 at 5 09 56 pm

After:

Screen Shot 2020-02-23 at 5 10 05 pm

---
PR by @klonos: https://github.com/backdrop/backdrop/pull/3084

### PR DESCRIPTION
**Pull request by klonos**
Fixes https://github.com/backdrop/backdrop-issues/issues/4318

Replaces https://github.com/backdrop/backdrop/pull/3083

### CODE REVIEW

File: core/modules/color/css/color.admin.css [hunk 1]
```diff
@@ -28,16 +28,32 @@ input[type=color] {
   margin-top: .5em;
   margin-bottom: .5em;
   clear: both;
-  vertical-align: middle;
 }
 
 /* Color inputs */
```
Comment (ghost) [inline core/modules/color/css/color.admin.css]: This no longer sits directly above `#system-theme-settings input[type=color] {` like it used to. It needs to be moved below the newly-added code.

File: core/modules/color/css/color.admin.css [hunk 2]
```diff
@@ -28,16 +28,32 @@ input[type=color] {
...
+
+/**
+  * Prevent color input labels from being crumbed up together, when there are
+  * some that wrap in multiple lines.
+  */
```
Comment (ghost) [inline core/modules/color/css/color.admin.css]: These lines have an extra space before the asterisk. They should line up with the first asterisk on the line above, not the second asterisk.

**Review by ghost — CHANGES_REQUESTED** [review]

File: core/modules/color/css/color.admin.css (hunk 2 above)
Comment (klonos) [reply core/modules/color/css/color.admin.css]: Oops 😅 ...fixed! 👍

File: core/modules/color/css/color.admin.css (hunk 1 above)
Comment (klonos) [reply core/modules/color/css/color.admin.css]: Sorry for being thick, but I don't understand what you mean @BWPanda. Can you explain further please?

Comment (ghost) [reply core/modules/color/css/color.admin.css]: The `Color inputs` comment originally sat right above the code to which it referred, not it's separated by the new code. So it should be moved down to sit above the `#system-theme-settings input[type=color] {` line again.

Comment (klonos) [reply core/modules/color/css/color.admin.css]: Sorry for taking this long ...I gotcha now. Fixing...

Comment (klonos) [reply core/modules/color/css/color.admin.css]: Fixed. Thanks for pointing this out @BWPanda 🙏

**Comment by klonos** [comment]
Getting some errors in php70 that seem unrelated to this PR:

```
Bootstrap: Page cache test (BootstrapPageCacheTestCase)

fail Background fetch copy served after granting access. bootstrap.test:308
fail Access denied page title shown. bootstrap.test:309
```

...let's try restarting the PR/tests.

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
Thanks! Merged into 1.x and 1.16.x.

**Pull request merged**